### PR TITLE
SDKS-874 Migrate Android SDK from bintray to maven

### PR DESCRIPTION
**Pull request by witrisna**

### CODE REVIEW

File: forgerock-authenticator/build.gradle [hunk 1]
```diff
@@ -6,11 +6,9 @@
  */
 
 apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
```
Comment (rodrigoareis) [inline forgerock-authenticator/build.gradle:9]: Update Copyright year

File: forgerock-core/build.gradle [hunk 2]
```diff
@@ -6,11 +6,9 @@
  */
 
 apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
```
Comment (rodrigoareis) [inline forgerock-core/build.gradle:9]: Update Copyright year

File: gradle.properties [hunk 3]
```diff
@@ -23,9 +23,6 @@ kotlin.code.style=official
 android.useAndroidX=true
 android.enableJetifier=true
 
-siteUrl = 'https://www.forgerock.com/'
```
Comment (rodrigoareis) [inline gradle.properties:26]: Update Copyright year

File: forgerock-core/build.gradle (hunk 2 above)
Comment (witrisna) [reply forgerock-core/build.gradle:9]: Updated

File: gradle.properties (hunk 3 above)
Comment (witrisna) [reply gradle.properties:26]: Updated

**Review by spetrov — APPROVED** [review]
Looks good to me! 👍 
Approved!

**Review by rodrigoareis — COMMENTED** [review]
Looks good to me

**Pull request merged**